exercises/practice/hamming/.docs/instructions.md

```diff
@@ -20,5 +20,4 @@ The Hamming Distance is useful for lots of things in science, not just biology,
 
 The Hamming distance is only defined for sequences of equal length, so
 an attempt to calculate it between sequences of different lengths should
-not work. The general handling of this situation (e.g., raising an
-exception vs returning a special value) may differ between languages.
+not work, returning an error.
```

Comment:
I missed that this PR still added a bit of text. This text should be remove and instead added to the other append document.

```suggestion
not work.
```